docs/src/main/asciidoc/security-openid-connect.adoc

```diff
@@ -8,15 +8,41 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 include::./attributes.adoc[]
 :toc:
 
-This guide demonstrates how to use Quarkus OpenID Connect (OIDC) Extension to protect your JAX-RS applications using Bearer Token Authorization where Bearer Tokens are issued by OpenID Connect and OAuth 2.0 compliant Authorization Servers such as https://www.keycloak.org[Keycloak].
+You can use the Quarkus OpenID Connect (OIDC) extension to secure your JAX-RS applications using Bearer Token Authorization. 
+The Bearer Tokens are issued by OIDC and OAuth 2.0 compliant authorization servers, such as https://www.keycloak.org[Keycloak]. 
 
-Bearer Token Authorization is the process of authorizing HTTP requests based on the existence and validity of a Bearer Token which provides valuable information to determine the subject of the call as well as whether an HTTP resource can be accessed.
+Bearer Token Authorization is the process of authorizing HTTP requests based on the existence and validity of a Bearer Token. 
+The Bearer Token provides information about the subject of the call which is used to determine whether or not an HTTP resource can be accessed.
 
-Please read the xref:security-openid-connect-web-authentication.adoc[Using OpenID Connect to Protect Web Applications] guide if you need to authenticate and authorize the users using OpenID Connect Authorization Code Flow.
+The following figures outline the Bearer Token Authorization mechanism in Quarkus:
 
-If you use Keycloak and Bearer tokens then also see the xref:security-keycloak-authorization.adoc[Using Keycloak to Centralize Authorization] guide.
+.Bearer Token Authorization mechanism in Quarkus
+image::security-bearer-token-authorization-mechanism-1.png[alt=Bearer Token Authorization, align=center]
+
+In the previous figure:
```

Comment:
```suggestion
Figure 1. Bearer Token Authorization mechanism in Quarkus
```

Reply:
This is how it is done at https://quarkus.io/version/main/guides/security-openid-connect-web-authentication (note it is in italics), so I think it has to be consistent; May be it should be `Diagram 1. ...`

Reply:
The `Figure 1....` is actually the caption for the image that we add to the diagram. 

How about changing it to `In the previous diagram`:

Reply:
@hmanwani-rh Hi, I'd rather have it consistent with the code flow doc, I'm not quite sure `in the previous...` is a typical way, may be we should just remove `In the previous...` and have no `Figure 1` and just start enumerating what is going on in the diagram ?

Reply:
@hmanwani-rh Thinking more about it, it does look better to me with this `Figure 1....` as we have 2 ones here and each one has a different description